### ISSUE TITLE
Dashboard UI rendering duplicate pipeline information

### ISSUE THREAD

**Issue by Diogomrol** [gocd/gocd#4775]
Hello there,

After upgrading to GoCD 18.5.0 I noticed that there is a bug in the new Dashboard that is duplicating/triplicating the pipeline info.

I can't really understand the root cause, but this started to happen after I unpaused a pipeline.

Screenshot:

![imagem](https://user-images.githubusercontent.com/9485620/40438420-f4d8faa8-5eaf-11e8-9dcf-1ad033bc2bd7.png)

Cheers,
Diogo Oliveira

**Comment by arvindsv** [comment]
A few questions I should have asked earlier:

1. Does it go away (fix itself) if you refresh?

2. Does it go away (fix itself) if you restart?

3. Did you just unpause and this happened? Any other steps you can think of? Was this pipeline created recently?

Thank you!

**Comment by Diogomrol** [comment]
@arvindsv, so answering your questions:

1) It does go away if I refresh the page
2) I didn't need to restart, because 1) fixed the problem
3) I was able to notice this behavior twice: on the first time I only unpaused the pipeline. On the second time, I unpaused the pipeline and then clicked on "Play".

Something that might be relevant: the pipeline is not new. However, it was the first time I ran it after upgrading from GoCD 17.10.0 to 18.5.0.

Thank you :)
